packages/devtools/lib/src/timeline/cpu_profile_model.dart

```diff
@@ -194,10 +201,40 @@ class CpuStackFrame extends TreeNode {
     return _inclusiveSampleCount;
   }
 
+  CpuStackFrame shallowCopy({bool copySampleCounts = false}) {
+    final copy = CpuStackFrame(
+      id: id,
+      name: name,
+      category: category,
+      url: url,
+      profileTime: profileTime,
+      profileSampleCount: profileSampleCount,
+    );
+    if (copySampleCounts) {
```

Comment:
why not always copy the sample counts?

Reply:
The sample counts are a reflection of the stack frame's relationship within the entire top-down profile. When copying the stack frame for a bottom up conversion, the inclusive sample count no longer applies. I changed the bool to be `resetInclusiveSampleCount`, and in that case, we just set `_inclusiveSampleCount` to null.